lib/config/fileconf.go

```diff
@@ -350,9 +350,13 @@ type Global struct {
 	// If omitted, the default will be used.
 	CASignatureAlgorithm *string `yaml:"ca_signature_algo,omitempty"`
 
+	// DEPRECATED: Prefer CAPins
 	// CAPin is the SKPI hash of the CA used to verify the Auth Server.
 	CAPin string `yaml:"ca_pin"`
 
+	// CAPins is the list of SKPI hashes fo the CAs used to verify Auth Servers.
+	CAPins []string `yaml:"ca_pins"`
+
```

Comment:
I think we have `apiutils.Strings`, which handles unmarshaling YAML as either a single string or a list of strings.
You can use that to keep the old field name.

Reply:
good point, switched that, nice to avoid a new field name here

Reply:
@awly @nklaassen I think this actually broke the case where no CA pin is provided, as `apiutils.Strings("")` unmarshals to `[]string{""}` instead of an empty slice.